pkg/kubelet/util/manager/watch_based_manager.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package manager
```

Comment:
For Kubelet, this is a non test related change. It would be nice to be a bit more careful on these.